xml/System.Windows.Forms/ToolStripLayoutStyle.xml

```diff
@@ -19,16 +19,16 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- You control <xref:System.Windows.Forms.ToolStrip> layout by choosing one of the members of <xref:System.Windows.Forms.ToolStripLayoutStyle> with the <xref:System.Windows.Forms.ToolStrip.LayoutStyle%2A> property.  
+ You control the layout of the <xref:System.Windows.Forms.ToolStrip> class and its derived classes by assigning a member of the <xref:System.Windows.Forms.ToolStripLayoutStyle> enumeration to the control's `LayoutStyle` property, such as the <xref:System.Windows.Forms.ToolStrip.LayoutStyle%2A> or <xref:System.Windows.Forms.StatusStrip> property.  
```

Comment:
the last part of the sentence isn't correct. Link to ToolStrip.LayoutStyle and StatusStrip.LayoutStyle using displayProperty=nameWithType perhaps?